### ISSUE TITLE
Breakage

### ISSUE THREAD

**Issue by annevk** [whatwg/spec-factory#1]
Currently there are the following issues. Since we don't run this often it's unclear whether they need to be resolved.
- [x] Compatibility uses `compatibility.bs` while its shortname is `compat`.
- [x] Makefile.template does not support  (do we really only need these for deploy, not review?)
   - [x] EXTRA_FILES
   - [x] POST_BUILD_STEP
- [x] Console
   * Different .editorconfig (indent_size = 2 rather than 1)
   * Different .gitignore
   * Has no .pr-preview.json
   * Custom Makefile (does not seem to be needed other than EXTRA_FILES though)
- [x] Encoding needs `/UnicodeData.txt` in `.gitignore`.
- [x] HTML is mostly incompatible as expected
   * Weird that it has indent_size = 2 and no max_line_length = 100.
   * No `.gitignore`. Probably because building is done elsewhere.
- [x] Streams
   * Minor .editorconfig mismatches again.
   * Needs `.gitignore additions
   * Has "post_processing" member in `.pr-preview.json`, but also seems to have some errors there (e.g., does not overwrite h1).
   * Makefile is custom as there's a node dependency.

Overall it works pretty well though (for 10 out of 15, and for the remaining 5 it still helps identifying issues) and ends up adding PULL_REQUEST_TEMPLATE.md to a bunch of places as well as making minor ordering changes to `.gitignore`.

**Comment by domenic** [comment]
> Compatibility uses compatibility.bs while its shortname is compat.

We could rename it. If we do it in a single commit history will still be there in Git, although it looks like the GitHub UI does not follow renames :(. So, maybe not worth it... I guess we could hard-code an exception into the tool, if we anticipate this being a long-term issue.

>  Console: Different .editorconfig (indent_size = 2 rather than 1)

We should fix the editorconfig and the spec to align.

> Console: Different .gitignore

This seems to be due to the `test/` and `reference-implementation/` subdirectories. However those have not been maintained, so we should just remove it. See also https://github.com/whatwg/console/issues/152.

The other divergences can be removed.

> Console: Has no .pr-preview.json

We should fix

> Console: Custom Makefile (does not seem to be needed other than EXTRA_FILES though)

We should align it. Although, we could copy some of its additions, which are reasonable practice:

```
SHELL=/bin/bash -o pipefail
.PHONY: local remote deploy review
```

(See [here](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) for the `.PHONY` business.)

> HTML: Weird that it has indent_size = 2 and no max_line_length = 100.

Oh wow, that's why my editor keeps resetting to indent_size 2!

>  Streams

I would like to remove Node from the build process at some point, and align on indent/line length.

**Comment by annevk** [comment]
It now works for everything except HTML and Streams. I'll still PR Streams with some local tweaks as overall it's an improvement there too.

**Comment by annevk** [comment]
https://github.com/whatwg/spec-factory/commit/a10cf795aadacd2f888373eccb01ad6443c5e950 excludes HTML/Streams. I suspect Streams can soon be enabled again and for HTML we probably want to to update a subset of files only.

**Comment by annevk** [comment]
Streams was fixed in #15.
